### ISSUE TITLE
A warrant canary is also a good idea for your website.

### ISSUE THREAD

**Issue by LeDechaine** [blueprint-freespeech/ricochet-refresh#144]
https://en.wikipedia.org/wiki/Warrant_canary

Example: https://silence.im/warrant/

**Comment by ASmith-** [comment]
I agree. These are dangerous times and the Tor Project team has been very active plugging and patching the Onion Network against all manner of State Sponsored attacks on the volunteer anonymous servers, the network itself and also the users with a wide variety of attack vectors. 

![Coal_Mine_Canary_dead](https://user-images.githubusercontent.com/8520894/185000775-e62d6dce-9be1-4cde-a137-c9455d2c4632.jpg)

![Coal_Mine_Canary-w-mask](https://user-images.githubusercontent.com/8520894/185000842-880bf705-ee46-4e1c-a792-61358016731e.jpg)
